source/reference/minio-mc/mc-support-diag.rst

```diff
@@ -23,7 +50,7 @@ The :mc-cmd:`mc support diag` command generates a health report for a MinIO depl
 
 .. end-mc-support-diag-desc
 
-For deployments registered with the MinIO subscription network (|subnet-short|), the command can automatically upload the health report for analysis.
+For deployments registered with the MinIO subscription network (|subnet-short|), the command can automatically upload the health report for analysis if you enable :mc-cmd:`~mc support callhome`.
```

Comment:
The command always uploads to SUBNET irrespective of callhome enabled or not.
If callhome is enabled, diagnostics data is uploaded automatically every 24hrs (without running any command).